src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Configurable Asset Picker
```

Comment:
Universal Editor Custom Asset Picker

Reply:
I was under impression that we agreed upon "Configurable asset picker?". Anyway, I will change name everywhere in this docs.

Reply:
Sorry.
It was the Universal Editor I wanted to emphasize.